server/aws-lsp-codewhisperer/src/language-server/netTransform/artifactManager.ts

```diff
@@ -199,10 +212,17 @@ export class ArtifactManager {
         })
     }
 
-    calculateMD5Sync(filePath: string): string {
+    async calculateMD5Sync(filePath: string): Promise<string> {
```

Comment:
1. Rename to `calculateMD5Async`
2. Do we need to `await` any calls to this function now that it is async?

Reply:
This is the only place